.github/workflows/release.yml

```diff
@@ -2,23 +2,38 @@ name: Docker Release
 
 on:
   push:
-    branches:
-      - master
     tags:
-      - v*
+      - 'v*.*.*'
 
-jobs:
+jobs:  
   release:
     runs-on: ubuntu-latest
     steps:
       -
         name: Checkout
         uses: actions/checkout@v2
       -
-        uses: docker/build-push-action@v1
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1      
```

Comment:
Is there anything different with the testcafe image that we need Docker Buildx?

Reply:
@christian-bromann this is needed by github actions, otherwise it fails to publish.